features/in_onderzoek.feature

```diff
@@ -597,6 +597,173 @@ Functionaliteit: in onderzoek
       En heeft de persoon met burgerservicenummer '555550001' GEEN 'naam.inOnderzoek' gegevens
       En heeft de persoon met burgerservicenummer '555550001' GEEN 'geboorte.inOnderzoek' gegevens
 
+    Scenario: de groep adres van de categorie verblijfplaats is in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
Waarom zijn er nieuwe scenario's toegevoegd en is de oude manier van gegeven stappen gebruikt? Dit betekent dus dat deze scenario's niet end-to-end werken

Reply:
Inmiddels komt deze manier van formuleren niet meer voor.